pygitguardian/client.py

```diff
@@ -605,3 +611,90 @@ def create_jwt(
             obj = load_detail(resp)
         obj.status_code = resp.status_code
         return obj
+
+    def compute_sca_files(
+        self,
+        files: List[str],
+        extra_headers: Optional[Dict[str, str]] = None,
+    ) -> Union[Detail, ComputeSCAFilesResult]:
+        if len(files) == 0:
+            result = ComputeSCAFilesResult(sca_files=[], potential_siblings=[])
+            result.status_code = 200
+            return result
+
+        response = self.post(
+            endpoint="sca/compute_sca_files/",
+            data={"files": files},
+            extra_headers=extra_headers,
+        )
+        result: Union[Detail, ComputeSCAFilesResult]
+        if is_ok(response):
+            result = ComputeSCAFilesResult.from_dict(response.json())
+        else:
+            result = load_detail(response)
+
+        result.status_code = response.status_code
+        return result
+
+    def sca_scan_directory(
+        self,
+        tar_file: bytes,
+        scan_parameters: SCAScanParameters,
+        extra_headers: Optional[Dict[str, str]] = None,
+    ) -> Union[Detail, SCAScanAllOutput]:
+        """
+        Generates tar archive associated with filenames and launches
```

Comment:
+1 for adding a docstring, but from what I understand, the function does not *generates* a tar archive, it expects the bytes of a tar archive to be passed to it, isn't it?

Reply:
You're right, that seems to be outdated ! Corrected